src/components/Events/ConferenceCard.js

```diff
@@ -30,54 +31,53 @@ const AnchorWrapper = ({ to, children }) =>
 
 const ConferenceCard = ({ event }) => {
   const {
-    status,
     date,
-    eventName,
-    eventLocation,
+    address,
+    status = 'TODO - status',
```

Comment:
You can work out status internally in the conference card. 

Using [isBefore](https://date-fns.org/v2.0.1/docs/isBefore) from dateFns we can just work out if its `Upcoming` or `Past`

```
const status = isBefore(new Date(), date)
```